.circleci/config.yml

```diff
@@ -37,7 +37,7 @@ jobs:
   cypress_integration_tests:
     working_directory: ~/app
     docker:
-      - image: cimg/node:14.15.3-browsers
+      - image: cimg/node:16.13-browsers
       - image: bitnami/redis:5.0
```

Comment:
could potentially look at using the common executor [here]( https://github.com/ministryofjustice/hmpps-template-typescript/blob/main/.circleci/config.yml#L101-L104), it might not "just work".